region/brinstar/pink/Big Pink.json

```diff
@@ -1195,6 +1202,11 @@
       ],
       "note": "If using Bombs, lure and kill the Reo. If using Spring Ball, it's important to release jump before reaching to peak of each jump to outrun the Reo."
     },
+    {
+      "link": [7, 7],
+      "name": "Crystal Flash",
+      "requires": ["h_canCrystalFlash"]
```

Comment:
7 can farm at 13 pretty quick.
CFing at 9, and also 1,  are the same situation as 7, but I think this strat can just go.